bin/codegen.dart

```diff
@@ -6,30 +6,32 @@ import 'templates/platformconstants.h.dart' as objc_header_template;
 import 'templates/platformconstants.java.dart' as java_template;
 import 'templates/platformconstants.m.dart' as objc_impl_template;
 
+typedef Template = String Function(Map<String, dynamic> context);
 
-typedef String Template(Map<String, dynamic> context);
-
-String projectRoot = "../";
+String projectRoot = '../';
 
 Map<Template, String> toGenerate = {
```

Comment:
Should be const

Reply:
f52bc782be2f579dd4b2931217a258a97957cea1